app/models/merchant.rb

```diff
@@ -2,7 +2,8 @@ class Merchant <ApplicationRecord
   has_many :items, dependent: :destroy
   has_many :item_orders, through: :items
   has_many :users
-
+  has_many :orders, through: :item_orders
```

Comment:
I don't see a `belongs_to` relationship in the `order` model. did I miss something?